bundles/org.openhab.binding.electroluxappliances/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,472 @@
+				<description>The appliance serial number.</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<thing-type id="electroluxwashingmachine">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="api"/>
+		</supported-bridge-type-refs>
+
+		<label>Electrolux Washing Machine</label>
+		<description>This thing represents the Electrolux Washing Machine.</description>
+
+		<channels>
+			<channel id="doorState" typeId="doorState"/>
+			<channel id="doorLock" typeId="doorLock"/>
+			<channel id="startTime" typeId="startTime"/>
+			<channel id="timeToEnd" typeId="timeToEnd"/>
+			<channel id="cyclePhase" typeId="cyclePhase"/>
+			<channel id="analogTemperature" typeId="analogTemperature"/>
+			<channel id="analogSpinSpeed" typeId="analogSpinSpeed"/>
+			<channel id="steamValue" typeId="steamValue"/>
+			<channel id="programsOrder" typeId="programsOrder"/>
+			<channel id="applianceState" typeId="applianceState"/>
+			<channel id="applianceMode" typeId="applianceMode"/>
+			<channel id="applianceUiSwVersion" typeId="applianceUiSwVersion"/>
+			<channel id="applianceTotalWorkingTime" typeId="applianceTotalWorkingTime"/>
+			<channel id="optisenseResult" typeId="optisenseResult"/>
+			<channel id="detergentExtradosage" typeId="detergentExtradosage"/>
+			<channel id="softenerExtradosage" typeId="softenerExtradosage"/>
+			<channel id="waterUsage" typeId="waterUsage"/>
+			<channel id="totalWashCyclesCount" typeId="totalWashCyclesCount"/>
+			<channel id="connectionState" typeId="connectionState"/>
+		</channels>
+
+		<properties>
+			<property name="vendor">Electrolux</property>
+			<property name="thingTypeVersion">1</property>
+		</properties>
+
+		<representation-property>serialNumber</representation-property>
+
+		<config-description>
+			<parameter name="serialNumber" type="text" required="true">
+				<label>Serial Number</label>
+				<description>The appliance serial number.</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+
+	<channel-type id="status">
+		<item-type>String</item-type>
+		<label>Current Status</label>
+		<description>Information on current status.</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>The measured temperature.</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%">
+		</state>
+	</channel-type>
+
+	<channel-type id="humidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>The measured humidity.</description>
+		<category>Humidity</category>
+		<state readOnly="true" min="0" max="100" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="tvoc">
+		<item-type>Number:Dimensionless</item-type>
+		<label>TVOC</label>
+		<description>The total Volatile Organic Compounds.</description>
+		<state readOnly="true" pattern="%d %unit%"/>
```

Comment:
Not sure if this is the right unit, but as it is dimenionless the unit has to be specified.
```suggestion
		<state readOnly="true" pattern="%d ppb"/>
```

Reply:
Fixed!